db/migrate/20211217150125_add_omniauth_to_users.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
Migrations almost always one time runners - no need to use `# frozen_string_literal: true`